qcodes_contrib_drivers/drivers/Tektronix/Keithley_6500.py

```diff
@@ -2,55 +2,154 @@
 from qcodes.instrument import InstrumentChannel
 from qcodes.utils.validators import Numbers
 from functools import partial
+from .Keithley_2000_Scan import Keithley_2000_Scan_Channel
 
 
 class Keithley_Sense(InstrumentChannel):
-    def __init__(self, parent: VisaInstrument, name: str, channel: str) -> None:
+    """
+    This is the class for a measurement channel, i.e. the quantity to be measured (e.g. resistance, voltage).
+    """
+    def __init__(self, parent: VisaInstrument, name: str, quantity: str) -> None:
```

Comment:
It usually not a good idea to rename an variable in a method definition even if the new name is better since this will break the use for any one else using the driver that passes the variable as a keyword argument.

Reply:
Thank you for the remark, I changed it back to the previous name "channel".